app/server/utils/transfer_limits.py

```diff
@@ -200,7 +207,8 @@ def __init__(self,
 
         self.application_filter = application_filter
         self.time_period_days = time_period_days
-        self.total_amount = total_amount
+        # TODO: Make LIMIT_EXCHANGE_RATE configurable per org
+        self.total_amount = int(total_amount * config.LIMIT_EXCHANGE_RATE) if total_amount else None
```

Comment:
`config.LIMIT_EXCHANGE_RATE` is only settable at a system level RN - should probably have this configurable per org once we do org level settings